config/jobs/kubernetes-sigs/kube-scheduler-simulator/kube-scheduler-simulator-presubmits-master.yaml

```
@@ -19,6 +19,7 @@ presubmits:
         - >
           make mod-download &&
           make tools &&
+          export PATH="${PATH}:${GOPATH}/bin"
```

Maybe `&&` is needed?
```suggestion
          export PATH="${PATH}:${GOPATH}/bin" &&
```

---

Ah, missed it again 😿 . Fixed, with the latest push.